app/app.go

```diff
@@ -466,7 +466,10 @@ func NewStrideApp(
 	stakeibcModule := stakeibcmodule.NewAppModule(appCodec, app.StakeibcKeeper, app.AccountKeeper, app.BankKeeper)
 	stakeibcIBCModule := stakeibcmodule.NewIBCModule(app.StakeibcKeeper)
 
-	app.InterchainqueryKeeper.SetCallbackHandler(stakeibcmoduletypes.ModuleName, app.StakeibcKeeper.CallbackHandler())
+	err := app.InterchainqueryKeeper.SetCallbackHandler(stakeibcmoduletypes.ModuleName, app.StakeibcKeeper.CallbackHandler())
+	if err != nil {
+		return nil
```

Comment:
should this be a panic instead?

Reply:
probably is worth panicing in app.go if these necessary setups fail. wdyt @asalzmann?

Reply:
I'm not sure - I think this goes back to the question of when to use panics vs. errors. In the original thread, the err advocate was saying app.go should not panic in this case, the argument for a panic was that something at a higher level will panic if you err.

I have a slight preference to keep this PR as minimal / lint focused as possible though, so leaving as an err for now (the app won't initialize if nil is returned)